scenes/test_object.txt

```diff
@@ -5,17 +5,13 @@
 
 *gosub test_start "get unmodified data"
 *gosub_scene cslib_object get "testobj" 1
-*gosub test_assert (testobj_testfield1 = "Calvin") "=> Expected - Calvin[n/]=> Received - ${testobj_testfield1}[n/]"
+*gosub test_assert (testobj_testfield1 = testobj_1_testfield1) "=> Expected - ${testobj_1_testfield1}[n/]=> Received - ${testobj_testfield1}[n/]"
```

Comment:
Can we make sure the 2nd field has been populated as well here?

Reply:
I'm getting a little confused as to what to do on these tests.
Could you help me out on the details?

Reply:
Let's look at these again when you've renamed the field attributes.

Reply:
Let's refactor this so that the new assert function gets only the two values, and assemble the message on its own